eng/common/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  EmulatorMsiUrl: "https://aka.ms/cosmosdb-emulator"
```

Comment:
We should also delete https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/templates/steps/cosmos-emulator.yml if no-one is actually using it yet.

Reply:
Couldn't find it being used anywhere so I've removed it.